belt-block/tests/mod.rs

```diff
@@ -0,0 +1,26 @@
+use cipher::{BlockDecrypt, BlockEncrypt, KeyInit};
+use hex_literal::hex;
+use belt_block::BeltBlock;
+
+/// Example vector from STB 34.101.31 (2020)
+#[test]
```

Comment:
Can you add a link to the cipher specification with test vectors used in this test? It can be in Russian or Belarusian.

Reply:
Also fixed, thanks!